pkgs/tools/networking/tubular/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "tubular";
+  version = "2022.2.1";
+
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-nK+YxHFy88mEQAcPt2gGz5JwA3G99YQf5wPjsTqCavc=";
+  };
+
+  vendorSha256 = null;
```

Comment:
Why no sha256?

Reply:
The dependencies are vendored (or shipped in tarball).